remoting/subsystem/src/main/java/org/jboss/as/remoting/Namespace.java

```diff
@@ -39,13 +39,17 @@ public enum Namespace {
     REMOTING_1_2("urn:jboss:domain:remoting:1.2"),
     REMOTING_2_0("urn:jboss:domain:remoting:2.0"),
     REMOTING_3_0("urn:jboss:domain:remoting:3.0"),
-    REMOTING_4_0("urn:jboss:domain:remoting:4.0")
+    REMOTING_4_0("urn:jboss:domain:remoting:4.0"),
+
+    REMOTING_5_0("urn:jboss:domain:remoting:5.0"),
```

Comment:
What's up with adding the schema 5 but then it's never registered?

Reply:
It is being registered in the RemotingExtension class.